main.js

```diff
@@ -0,0 +1,102 @@
+var ShowShows = { 
+    init: function() { 
+        $('#LocationForm').on('submit', function(e) { 
+            e.preventDefault(); 
+
+            ShowShows.getShows($('#EventLocation').val());
+        }); 
+    }, 
+
+    getShows: function(location) { 
+
+
+        $.ajax({
+            url: "http://api.bandsintown.com/events/search.json?location=" + location + "&radius=10&page=1&per_page=10&app_id=Marianne_JS_APP",
+            dataType: "jsonp", 
+
+        }).done(function(events) {
+            ShowShows.mapCenter = ShowShows.calcMapCenter(events); 
+            ShowShows.initMap(); 
+
+            events.forEach(function(event) { 
+                ShowShows.createHtml(event);  
+            }); 
+
+        });
+    }, 
+
+    createHtml: function(event) { 
```

Comment:
I might make it more clear what HTML is being created, by calling this `displayEvent` or something.